notebooks/concepts/SpecViz.ipynb

```diff
@@ -0,0 +1,138 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Suppress warnings\n",
+    "import warnings\n",
+    "\n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'/home/duy/.astropy/cache/download/py3/b2b3e0df0f240582d271bb14dc59311c'"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# This file is originally from https://data.sdss.org/sas/dr14/manga/spectro/redux/v2_1_2/7495/stack/manga-7495-12704-LOGCUBE.fits.gz\n",
```

Comment:
This looks like a typo to me - the spectrum loaded below is not a MANGA spectrum (which is a cube), but rather the example SDSS spectrum we use in specviz (which is a 1d spectrum).  That's *good*, because the 1d spectrum is the right thing, but ithis comment is now misleading.

Reply:
Thanks for the catch!